addon/ng2/blueprints/enum/files/__path__/__name__.spec.ts

```diff
@@ -0,0 +1,14 @@
+import {
+  describe,
+  ddescribe,
+  expect,
+  iit,
+  it
+} from '@angular/core/testing';
+import {<%= classifiedModuleName %>} from './<%= fileName %>';
+
+describe('<%= classifiedModuleName %>', () => {
+  it('should properly exist', () => {
+    expect(<%= classifiedModuleName %>).toBeTruthy();
+  });
+});
```

Comment:
I agree. I just thought, there have to be some tests, as in the other blueprints. But you're right for enums it doesn't make any sense.